README.md

```diff
@@ -88,27 +88,27 @@ For API documentation go to
 
  * [Realm Dart API Docs](https://pub.dev/documentation/realm_dart/latest/)
 
-For a complete documentation go to [Realm Flutter and Dart SDK Docs](https://docs.mongodb.com/realm/sdk/flutter/).
+Use [realm](https://pub.dev/packages/realm) package for Flutter and [realm_dart](https://pub.dev/packages/realm_dart) package for Dart applications.
 
+For complete documentation of the SDKs, go to the [Realm SDK documentation](https://docs.mongodb.com/realm/sdk/flutter/).
 
-## Limitations
+If you are using the Realm SDK for the first time, refer to the [Quick Start documentation](https://www.mongodb.com/docs/realm/sdk/flutter/quick-start/).
 
-* It provides the functionality for creating, retrieving, querying, sorting, filtering, updating Realm objects.
+To learn more about using Realm with Atlas App Services and Device Sync, refer to the following Realm SDK documentation:
 
-* Migrations are not supported yet.
-
-    If you change your data models often and receive a migration exception be sure to delete the old `default.realm` file in your application directory. It will get recreated with the new schema the next time the Realm is opened.
+- [App Services Overview](https://www.mongodb.com/docs/realm/sdk/flutter/app-services/)
+- [Device Sync Overview](https://www.mongodb.com/docs/realm/sdk/flutter/sync/)
 
 # Realm Flutter SDK
 
-The Realm Flutter package name is `realm`
+The Realm Flutter package name is [realm](https://pub.dev/packages/realm).
```

Comment:
We mention this in the beginning. Should we have this here as well?

Reply:
It is good to have the link here in the relevant section. But you are right that we already mention the names above. Is it OK, to change the text like this:
"Realm Flutter package is published to [realm](https://pub.dev/packages/realm)."?
The same for Dart.